modules/running-network-verification-manually-cli.adoc

```diff
@@ -19,18 +19,42 @@ When you run the network verification, you can specify a set of VPC subnet IDs o
 
 .Procedure
 
-* Verify the network configuration by using one of the following methods: 
+* Verify the network configuration by using one of the following methods:
 ** Verify the network configuration by specifying the cluster name. The subnet IDs are automatically detected:
 +
 [source,terminal]
 ----
-$ rosa verify network -c <cluster_name> <1>
+$ rosa verify network --cluster <cluster_name> <1>
 ----
 <1> Replace `<cluster_name>` with the name of your cluster.
 +
 .Example output
 [source,terminal]
 ----
+I: Verifying the following subnet IDs are configured correctly: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
+I: subnet-0b761d44d3d9a4663: pending
+I: subnet-0f87f640e56934cbc: passed
+----
+*** Ensure that verification to all subnets has been completed.
++
+[source,terminal]
+----
+$ rosa verify network --watch <1>
+                      --status-only <2>
+                      --region us-east-1 <3>
+                      --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc <4>
+----
+<1> The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
+<2> The `status-only` flag does not trigger a run of network verification but returns the current state that is pending, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
+<3> Use a specific AWS region, which overrides the _AWS_REGION_ environment variable.
+<4> Enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the command exits with error code one (1) and no subnets are checked.
++
+.Example output
+[source,terminal]
+----
+I: Checking the status of the following subnet IDs: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
+I: subnet-0f87f640e56934cbc: passed
+I: subnet-0b761d44d3d9a4663: passed
 I: ✓ Network verification successful
```

Comment:
I don't see this info "I: ✓ Network verification successful" after I run the command

```
./rosa verify network  --status-only --region us-west-2 --subnet-ids subnet-082410d8fb3d95195,subnet-0fb407a4dc8974e87,subnet-0700cf0d732aef010,subnet-0febb5c2620e3548e,subnet-0ba62f961cac934f9,subnet-0f8ac35d20f1d659d
I: Checking the status of the following subnet IDs: [subnet-082410d8fb3d95195 subnet-0fb407a4dc8974e87 subnet-0700cf0d732aef010 subnet-0febb5c2620e3548e subnet-0ba62f961cac934f9 subnet-0f8ac35d20f1d659d]
I: subnet-082410d8fb3d95195: passed
I: subnet-0fb407a4dc8974e87: passed
I: subnet-0700cf0d732aef010: passed
I: subnet-0febb5c2620e3548e: passed
I: subnet-0ba62f961cac934f9: passed
I: subnet-0f8ac35d20f1d659d: passed
```

Reply:
Thanks, I will update this.

> I don't see this info "I: ✓ Network verification successful" after I run the command
> 
> ```
> ./rosa verify network  --status-only --region us-west-2 --subnet-ids subnet-082410d8fb3d95195,subnet-0fb407a4dc8974e87,subnet-0700cf0d732aef010,subnet-0febb5c2620e3548e,subnet-0ba62f961cac934f9,subnet-0f8ac35d20f1d659d
> I: Checking the status of the following subnet IDs: [subnet-082410d8fb3d95195 subnet-0fb407a4dc8974e87 subnet-0700cf0d732aef010 subnet-0febb5c2620e3548e subnet-0ba62f961cac934f9 subnet-0f8ac35d20f1d659d]
> I: subnet-082410d8fb3d95195: passed
> I: subnet-0fb407a4dc8974e87: passed
> I: subnet-0700cf0d732aef010: passed
> I: subnet-0febb5c2620e3548e: passed
> I: subnet-0ba62f961cac934f9: passed
> I: subnet-0f8ac35d20f1d659d: passed
> ```

Thanks I will change this.